src/components/breadcrumbs/_breadcrumbs_simplified.scss

```diff
@@ -77,9 +86,8 @@
   transform: translateY(-1px) rotate(15deg);
   background: $ouiColorLightShade;
   color: $ouiColorDisabledText;
-}
 
-.euiLink.euiSimplifiedBreadcrumb {
-  line-height: inherit;
-  font-weight: inherit;
-}
\ No newline at end of file
+  &:last-child {
```

Comment:
does it still show up when the last child is not a leaf node?

Reply:
need to address this